src/index.ts

```diff
@@ -149,6 +167,18 @@ async function initialBuild(): Promise<void> {
             })
         )
     );
+
+    // Minify js files with terser if in production.
+    if (IS_PRODUCTION_MODE) {
```

Comment:
I'd like to add this opt-out mechanism so that users can choose a different way to minify if terser doesn't work.

```suggestion
    if (IS_PRODUCTION_MODE && !process.argv.includes('--no-minify')) {
```